site/components/pages/Redesign/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { NextPage } from "next";
+
+import {
+  PageWrap,
+  HeaderDesktop,
+  Footer,
+  NavItemDesktop,
+  HeaderMobile,
+  NavItemMobile,
+  ButtonPrimary,
+} from "@klimadao/lib/components";
+
+import styles from "./index.module.css";
+import { PageHead } from "components/PageHead";
+import { IS_PRODUCTION } from "lib/constants";
+import { urls } from "@klimadao/lib/constants";
+
+export interface Props {
+  treasuryBalance: number;
+  stakingAPY: number;
+  price: number;
+}
+
+export const Home: NextPage<Props> = (props) => {
+  return (
+    <>
+      <PageHead
+        production={IS_PRODUCTION}
+        title="KlimaDAO"
+        mediaTitle="KlimaDAO"
+        metaDescription="Drive climate action and earn rewards with a carbon-backed digital currency."
+        mediaImageSrc="/og-media.jpg"
+      />
+      <PageWrap>
+        <HeaderDesktop
+          buttons={[
+            <ButtonPrimary
+              key="Connect Wallet"
+              label="Connect Wallet"
+              onClick={() => console.log("Do we want this here?")}
```

Comment:
no, no wallet connection anywhere in `site`. I brought this up a few weeks ago but designs never got updated, sorry!

All wallet interactions should be contained in the `app` because `ethers.js` is beefy and because the `app` can be exported as a standalone plain html app. But the rest of the website can not.

Reply:
Thanks for the update. Changed to "Enter App" as stated in the updated designs